tagilmo/utils/vereya_wrapper.py

```diff
@@ -269,15 +269,25 @@ def isItemsListAvailable(self, nAgent=0):
     def isRecipesListAvailable(self, nAgent=0):
         return not self.observe[nAgent] is None and 'recipes' in self.observe[nAgent]
 
+    def isTriplesListAvailable(self, nAgent=0):
+        return not self.observe[nAgent] is None and 'block_item_tool_triple' in self.observe[nAgent]
+
     def getItemList(self, nAgent=0):
         if self.isItemsListAvailable(nAgent):
             observations = self.observe[nAgent]
             item_list = observations['item_list']
-            self.sendCommand('item_list off')
             return item_list
         else:
             return None
 
+    def getBlocksDropsList(self, nAgent=0):
+        if self.isTriplesListAvailable(nAgent):
+            observations = self.observe[nAgent]
+            triples = observations['block_item_tool_triple']
+            return triples
+        else:
+            return None
```

Comment:
this code can be simplified, no need for additional functions

```suggestion
        observations = self.observe[nAgent]
        if observations is not None:
            triples = observations.get('block_item_tool_triple', None)
            return triples
```

Reply:
but observation could be not None but still won't contain block_item_tool_triple. Point of the additional function to check this particular thing.

Reply:
@noskill  you can check now, i've simplified and united some functions into one to remove code duplication. Your suggested change currently gives me an error since, as i said, observation in this point can be not None but won't contain necessary field.